--- origsrc/audioresample/Makefile.am	2004-03-12 19:37:27.000000000 -0600
+++ src/audioresample/Makefile.am	2006-08-06 18:40:42.665625000 -0500
@@ -4,3 +4,4 @@
 pkgconfigdir = $(libdir)/pkgconfig
 pkgconfig_DATA = audioresample-@AUDIORESAMPLE_MAJORMINOR@.pc
 
+ACLOCAL_AMFLAGS = -I m4
--- origsrc/audioresample/audioresample/Makefile.am	2005-05-28 21:30:10.000000000 -0500
+++ src/audioresample/audioresample/Makefile.am	2006-08-06 18:40:42.665625000 -0500
@@ -15,11 +15,11 @@
 	buffer.h
 libaudioresample_@AUDIORESAMPLE_MAJORMINOR@_la_LIBADD = \
 	$(LIBOIL_LIBS) \
-	$(GLIB_LIBS)
+	$(GLIB_LIBS) -lm
 libaudioresample_@AUDIORESAMPLE_MAJORMINOR@_la_CFLAGS = \
 	$(AUDIORESAMPLE_CFLAGS) $(GLIB_CFLAGS)
 libaudioresample_@AUDIORESAMPLE_MAJORMINOR@_la_LDFLAGS = \
-	-lm
+	-no-undefined
 
 libaudioresampleincludedir = $(includedir)/audioresample-@AUDIORESAMPLE_MAJORMINOR@/audioresample
 libaudioresampleinclude_HEADERS = \
--- origsrc/audioresample/audioresample-0.1.pc.in	2004-12-15 17:30:04.000000000 -0600
+++ src/audioresample/audioresample-0.1.pc.in	2006-08-06 18:40:42.681250000 -0500
@@ -8,7 +8,7 @@
 Name: audioresample-@AUDIORESAMPLE_MAJORMINOR@
 Description: Audio resampling library
 Version: @VERSION@
-Requires: liboil-0.2
+Requires: liboil-0.3
 Libs: -L${libdir} -laudioresample-@AUDIORESAMPLE_MAJORMINOR@
 Cflags: -I${includedir}/audioresample-@AUDIORESAMPLE_MAJORMINOR@
 
